validator/project_validator.go

```diff
@@ -139,6 +139,7 @@ var projectErrorValidators = []projectValidator{
 	validateHostCreates,
 	validateDuplicateBVTasks,
 	validateGenerateTasks,
+	validateModuleUsageInGitGetProject,
```

Comment:
I think validators that warn should be in projectWarningValidators and errors should be in this list. cc @hadjri to verify.
(I wonder if we should make some sort of linter or test to verify that we keep things in that state? Seems easy to miss and then we'll have to fix everything again)

Reply:
I think so! I forgot to move it to the warnings rather than the errors check, I'll move it and retest to make sure it still propagates

Reply:
Yup can confirm this would belong in the warning list.

Reply:
Created https://jira.mongodb.org/browse/DEVPROD-3996 re your suggestion @ablack12